contracts/core/Core.sol

```diff
@@ -33,7 +34,8 @@ import { IVault } from "./interfaces/IVault.sol";
  * creating Sets, as well as all collateral flows throughout the system.
  */
 contract Core is
-    Ownable
+    CoreState,
```

Comment:
is this necessary? `CoreInternal` also inherits from it so won't it be enough to just say `is CoreInternal`?

Reply:
+1